test_scripts/Policies/Related_HMI_API/178_ATF_OnAppPermissionConsent_without_appID.lua

```diff
@@ -28,11 +28,11 @@ local commonFunctions = require ('user_modules/shared_testcases/commonFunctions'
 local commonSteps = require('user_modules/shared_testcases/commonSteps')
 local testCasesForPolicyTableSnapshot = require('user_modules/shared_testcases/testCasesForPolicyTableSnapshot')
 local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local testCasesForPolicyAppIdManagament = require("user_modules/shared_testcases/testCasesForPolicyAppIdManagament")
```

Comment:
@GetmanetsIrina `AppID` needs to removed from `SDL.GetListOfPermissions` and `SDL.OnAppPermissionConsent` in order to conform the description of the test

Reply:
@dboltovskyi Done in e0df32268be3a70e2763967f3cc9491797637a76